### PR TITLE
Stupid Unneccessary and Hacky feature.

### PR DESCRIPTION
**Pull request by Cruix**
### Intent of your Pull Request

:cl:
rscadd: You can now wear buckets on your head.
/:cl:
You can now wear buckets on your head. They blind you, make you immune to flashes, hide your identity, and provide the same armor as a security helmet. When you put it on, any reagents it contains will be splashed over you. I tried to delete it whenever you put it on your head and replace it with a proper /obj/item/clothing/head, but that didn't work, so I just gave it a compatible interface instead.

I could be coding something useful but instead I code this.

untitled

### CODE REVIEW

**Comment by FluffySurvivor** [comment]
:+1: :+1: :+1:
dont let yourself down

**Comment by oisin100** [comment]
10/10
:+1:

**Comment by Super3222** [comment]
:+1:

Slap an Adam Likes on it, because you know he will.

**Comment by Cruix** [comment]
It was actually his idea.

**Comment by X-TheDark** [comment]
Eh, why not?

**Comment by ToGWtF** [comment]
so i can add bedsheet ghosts back to the game yes

File: code/modules/reagents/reagent_containers/glass.dm [hunk 1]
```diff
@@ -287,3 +293,19 @@
...
+/obj/item/weapon/reagent_containers/glass/bucket/equipped(mob/user, slot)
+	if(slot == slot_head)
+		user.visible_message("<span class='danger'>The contents of [src] dump onto [user]!</span>")
+		var/R
+		if(reagents)
+			for(var/datum/reagent/A in reagents.reagent_list)
+				R += A.id + " ("
+				R += num2text(A.volume) + "),"
+		reagents.reaction(user, TOUCH)
+		user.attack_log += "Had a bucket containing [R] dumped on them."
```
Comment (george99g) [inline code/modules/reagents/reagent_containers/glass.dm]: Shouldn't the attack log say who dumped it onto them?
Seems like something an admin would want to know.

**Comment by ShadowDeath6** [comment]
> try to quick equip bucket into backpack
> put it on head

:+1:

**Comment by ghost** [comment]
Make it not blind but just make you nearsighted

**Comment by ShadowDeath6** [comment]
I didn't realize putting a bucket over your head which completely blocks all vision is the same as wearing welding goggles

Also new meta is to put a bucket filled with water on your head whenever you're on fire

**Comment by Cruix** [comment]
> Make it not blind but just make you nearsighted

Sadly I cannot confer 1337 haxor coding abilities with a helmet equip item.

> Also new meta is to put a bucket filled with water on your head whenever you're on fire

I mean you could already do this with a bucket on harm intent but yeah I can see how it's op now.

**Comment by ghost** [comment]
put a bucket of synthflesh on your head when in danger

tactical bucket

**Comment by X-TheDark** [comment]
Or put a bucket of Sulfuric Acid over someone. **HONK**

**Comment by george99g** [comment]
> Or put a bucket of Sulfuric Acid over someone. HONK

But you could already do this...

**Comment by DavidEdwards** [comment]
I love it, I really do. People might see functionality like this as a waste of time, but this is really ingenious. It gives players more silly ways to get things done.

**Pull request merged**